wordpress-mysql-replication/mainTemplate.json

```diff
@@ -0,0 +1,384 @@
+{
```

Comment:
This will be integrated with marketplace.  If this repo only takes azuredeploy.json, and marketplace only takes mainTemplate.json, what would you recommend other than having 2 duplicate files?